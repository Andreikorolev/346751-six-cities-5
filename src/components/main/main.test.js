import React from "react";
import renderer from "react-test-renderer";
import Main from "./main";

describe(`Main render`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(
          <Main
            renderClassName={() => {}}
          >
            <React.Fragment/>
          </Main>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
