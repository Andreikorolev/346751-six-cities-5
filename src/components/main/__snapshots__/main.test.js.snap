// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main render should render correctly 1`] = `
<main
  className="page__main undefined"
/>
`;
